src/atoms/Link/Link.stories.js

```diff
@@ -10,12 +10,21 @@ const sizeOptions = {
   large: 'large',
 };
 
+const targetOptions = {
+  blank: '_blank',
+  self: '_self',
+};
+
 storiesOf('Link', module)
   .addParameters({
     component: Link,
   })
   .add('default', () => (
-    <Link size={select('Size', sizeOptions, 'large')}>
+    <Link
+      size={select('Size', sizeOptions, 'large')}
```

Comment:
We haven't implemented `rel` attribute as well. and also the values for targetoptions are not just 2.
Refer the whole `<a/>` tag here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

Reply:
I never saw anyone using other target options, so I didn't add.  But I will add it now.